docs/components/CreateNftSection.tsx

```diff
@@ -1,8 +1,111 @@
+import { useState } from "react";
+import classNames from "classnames";
+import { Formik, Field, useFormikContext } from "formik";
 
 export const CreateNftSection = () => {
   return (
     <div>
-      <b>Hi this is the create NFT section</b>
+      <Formik
+        initialValues={{ name: "", image: null }}
+        onSubmit={(values) => console.log(values)}
+      >
+        {(props) => (
+          <form onSubmit={props.handleSubmit}>
```

Comment:
```suggestion
          <Form>
```

Formik handles onSubmit so we don't need to do that here